Framework/DataObjects/src/SplittersWorkspace.cpp

```diff
@@ -64,6 +64,16 @@ bool SplittersWorkspace::removeSplitter(size_t index) {
   return removed;
 }
 
+Kernel::TimeSplitter SplittersWorkspace::convertToTimeSplitter() {
```

Comment:
Please add doxygen to explain what this does and the effect that if there are overlaps, later rows will overwrite earlier rows. This is a subtle behavior that should be documented. It would be even better if this method logged a warning when there is an overlap.